src/py21cmfast/wrapper.py

```diff
@@ -1459,6 +1459,7 @@ def __init__(
         perturb: PerturbedField,
         ib: IonizedBox,
         bt: BrightnessTemp,
+        st: [TsBox, None],
```

Comment:
Why is it in this format rather than just TsBox. Is it because of some convention? (just curious)

Reply:
this indicates it can be a `TsBox` or None (since this is optional). Actually, I should set it to None by default...

Reply:
Made it None by default.